proposals/jorydotcom-PROJECT_PROGRESSION/PROJECT_PROGRESSION.md

```diff
@@ -0,0 +1,116 @@
+## I. Overview
+This governance policy describes how an open source project can formally join the Foundation via the [Project Proposal Process](). It describes the [Stages]() a project may be admitted under and what the criteria and expectations are for a given stage, as well as the acceptance criteria for a project to move from one stage to another. It also describes the [Annual Review Process]() through which those changes will be evaluated and made. 
+
+Benefits of the project progression process include:
+* Promotion and public visibility of work that can add value to the _$FOUNDATION_ mission and build the ingredients of a successful project
+* Deepened alignment with existing projects
+* Mentorship and support 
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to _$FOUNDATION_ legal, code of conduct and IP Policy requirements
```

Comment:
I would state "access to the travel fund and other resources the foundation could provide to projects".

Reply:
For all of the above bullet points:

- Is this well-defined?
- How does $foundation plan to accomplish this?

Maybe the answers to these questions don't belong here, but they're important questions (and not just because I have them!)

Reply:
I'd agree that it is important that they understand why they would want to join.  It might be best to have a specific document "Why the in the world would I want to join the X foundation?"

Reply:
I'm going to take a pass at some introductory prose that gets to this question & creates a spot we can answer it more directly.